src/main/java/org/fundacionjala/at15/katas/pokerhands/adhemar/PokerHand.java

```diff
@@ -0,0 +1,21 @@
+package org.fundacionjala.at15.katas.pokerhands.adhemar;
+
+public class PokerHand extends Pattern {
```

Comment:
Oh, I see the point of PokerHand. Still, if we would like to add more patterns in the future, is this extensible enough?

Reply:
Actually, It isn't extensible enough, I'd have to modify Pattern class directly, but I'm going to work on it

Reply:
Now I can add more patterns if I want only creating a new class which extends from Pattern, @J0c4 done